apps/dcos_l4lb/src/dcos_l4lb_mgr.erl

```diff
@@ -623,6 +625,19 @@ bes_port_mappings(PMs, Protocol, AgentIP, BEs) ->
                 {BEAgentIP, {BEIP, BEPort}}
         end, BEs).
 
+%%%===================================================================
+%%% GC funtions
+%%%===================================================================
+
+-spec(handle_gc(state()) -> state()).
+handle_gc(#state{ipvs_mgr=IPVSMgr, route_mgr=RouteMgr,
```

Comment:
curious: why we have to explicitly garbage collect these structures instead of `hibernate`?

Reply:
It's the same reason as here: https://github.com/dcos/dcos-net/pull/156#discussion_r293197818. `dcos_l4lb_mgr` does reconciliation every 30 seconds, to avoid any other logic or gc timer, it explicitly garbage collect its child processes.

Reply:
Thanks